molo/core/templatetags/core_tags.py

```diff
@@ -36,12 +40,17 @@ def get_pages(context, queryset, locale):
     show_only_translated_pages = SiteSettings.for_site(
         request.site).show_only_translated_pages
     pages = []
-    for page in queryset:
-        translation = page.get_translation_for(locale, request.site)
-        if translation and translation.live:
-            pages.append(translation)
-        elif page.live and not show_only_translated_pages:
-            pages.append(page)
+    if settings.USE_NEW_TRANSLATIONS:
```

Comment:
there should be an else for` if settings.USE_NEW_TRANSLATIONS:`

Reply:
there is? it's on line 47